buildenv/buildenv.nix

```diff
@@ -0,0 +1,278 @@
+{
+  activationScripts_storePath ? "@activationScripts@",
+  defaultEnvrc_storePath ? "@defaultEnvrc@",
+  coreutils_storePath ? "@coreutils@",
+  floxBuildenv_storePath ? "@out@",
+  manifest,
+  name ? "environment",
+  serviceConfigYaml ? "",
```

Comment:
suggestion (nb): for consistency with other "optionals" use `null`

Reply:
The reason I did it this way is because that makes it more annoying in `buildenv.bash` to omit the argument when not provided, but OK ... fixed in commit 4cecefe33287f51890e3bc7f1bb3df49d9fea2ae